(node:3183256) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-40602',
  containerIp: '10.0.3.130',
  mitmPort: 40602,
  autoAccessFixed: 1
}
2024-11-06 08:12:35.692 - [Info] MITM Version: 2
2024-11-06 08:12:35.695 - [Info] Auto Access Enabled: true
2024-11-06 08:12:35.695 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-06 08:12:35.723 - [Info] SSH man-in-the-middle server for 10.0.3.130 listening on 10.0.3.1:40602
2024-11-06 08:13:25.700 - [Debug] [Connection] Attacker connected: 190.145.200.210 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 08:13:26.449 - [Debug] [Auth] Attacker 190.145.200.210 trying to authenticate with 'password'
2024-11-06 08:13:26.451 - [Debug] [Auto Access] Attacker: 190.145.200.210, Threshold: 1, Attempts: 1
2024-11-06 08:13:26.451 - [Debug] [Auto Access] Compromising the honeypot
2024-11-06 08:13:26.451 - [Debug] [Auto Access] Adding the following credentials: 'test:test1234'
2024-11-06 08:13:26.867 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-06 08:13:26.877 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-06 08:13:26.878 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.130
2024-11-06 08:13:27.002 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-06 08:13:27.334 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-06 08:13:27.809 - [Debug] [Connection] Attacker connected: 190.145.200.210 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 08:13:28.248 - [Debug] [Auth] Attacker 190.145.200.210 trying to authenticate with 'password'
2024-11-06 08:13:28.249 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-06 08:13:28.250 - [Debug] [Auth] Attacker: 190.145.200.210 has so far made 1 attempts. Remaining: 5 attempts
2024-11-06 08:13:28.416 - [Debug] [Connection] Attacker closed the connection
2024-11-06 08:13:28.478 - [Debug] [Connection] Attacker connected: 190.145.200.210 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-06 08:13:28.896 - [Debug] [Auth] Attacker 190.145.200.210 trying to authenticate with 'password'
2024-11-06 08:13:28.896 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2024-11-06 08:13:28.897 - [Debug] [Auth] Attacker: 190.145.200.210 has so far made 1 attempts. Remaining: 5 attempts
2024-11-06 08:13:29.062 - [Debug] [Connection] Attacker closed the connection
2024-11-06 08:13:29.186 - [Debug] [Connection] Attacker closed connection
2024-11-06 08:13:29.199 - [Debug] [LXC] Container's OpenSSH server ended connection
2024-11-06 08:13:29.200 - [Debug] [LXC] Container's OpenSSH server closed connection
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
